public/js/utils/redux/middleware/log.js

```diff
@@ -9,7 +9,7 @@
  */
 function log({ dispatch, getState }) {
   return next => action => {
-    console.log("[DISPATCH]", JSON.stringify(action, null, 2));
+    console.log(`[DISPATCH ${action.type}]`, action);
```

Comment:
> CI: printing full logs
That's a good point

> what's your workflow

I like to be able to scroll the log in question. For example, if i'm looking at the source that's selected when the debugger pauses, I want to scroll to the pause event, and then go to the next select source action. This might be the 10th action and the pause action is so large that it can be tough to find. 

Ideally, this wouldn't be as much of a question because there would be a better redux action log :/